crates/turborepo-lib/src/turbo_json/mod.rs

```diff
@@ -80,19 +89,27 @@ pub struct RawTurboJson {
 
 #[derive(Serialize, Default, Debug, PartialEq, Clone)]
 #[serde(transparent)]
-pub struct Pipeline(BTreeMap<TaskName<'static>, RawTaskDefinition>);
+pub struct Pipeline(BTreeMap<TaskName<'static>, PipelineEntry>);
+
+#[derive(Serialize, Default, Debug, PartialEq, Clone)]
+#[serde(transparent)]
+pub struct PipelineEntry {
```

Comment:
Is there a reason why this is necessary and we can't just use `Spanned<RawTaskDefinition>`? Easily could be, but I'm curious as to what's blocking it.

Reply:
Eh not particularly anymore. That was an artifact of something else that I tried